Packs/Tanium/Integrations/Tanium/Tanium.yml

```diff
@@ -759,7 +759,7 @@ script:
     - contextPath: Tanium.Actions.package_spec.name
       description: Name of the package that was deployed
       type: string
-  dockerimage: demisto/pytan
+  dockerimage: demisto/pytan:latest
```

Comment:
let's update the image to have numeric value

Reply:
This specific docker is kind of hard to update because of the pytan package - this is the base package for using Tanium.

It is not simply imported but patched in and it does not exist in PyPi.
![image](https://user-images.githubusercontent.com/50295826/81939281-d110ea80-95fe-11ea-9e2c-faf6ed10c3d4.png)

Considering we have Tanium v2 already out, I am not sure we want to pursue this specific docker image.